### PR TITLE
Updated instructions and formatting in README.

### PR DESCRIPTION
**Pull request by Mikunj**
Just updated README to look a bit nicer and to have more specific installation instructions.
Let me know if i need to add anything else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+
+1. Run `yarn install`
+2. Run `react-native link`
+3. Because of weird bug, react native does not have btoa and atob functions, but it is needed in auth library, for that we need to import btoa and atob in that library. (this is temporary solution, I will create PR to support react-native to that library). For now, you can run the app by enabling Remote debug mode, and app will work as it should.
 
-Debugging:
+## Debugging:
 - Install [react-native-debugger](https://github.com/jhen0409/react-native-debugger/releases)
-- before running, `yarn run debug`
+- Run `yarn run debug`
```
Comment (joshmadewell) [inline README.md]: just `yarn debug` is needed

File: README.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
-1. Run yarn install
-2. Run react-native link
-3. Because of weird bug, react native does not have btoa and atob functions, but it is needed in auth library, for that we need to import btoa and atob in that library. (this is temporary solution, I will create PR to support react-native to that library). For now, you can run the app by enabling Remote debug mode, and app will work as it should.
+## Installation
+
+### Prerequsites
+- [yarn](https://yarnpkg.com/lang/en/docs/install/)
+- [watchman](https://facebook.github.io/watchman/docs/install.html)
+- react-native-cli
+    - `npm install -g react-native-cli`
```
Comment (joshmadewell) [inline README.md]: `yarn global add react-native-cli` since we're using yarn anyway :D

File: README.md (hunk 1 above)
Comment (Mikunj) [reply README.md]: Oh alright i'll fix that up quick :)

**Comment by Mikunj** [comment]
Updated with the requested changes

**Pull request merged**